sechub-doc/src/docs/asciidoc/documents/shared/configuration/sechub_config_example8_web_scan_openapi_with_data_reference.json

```diff
@@ -12,7 +12,8 @@
     "url" : "https://productfailure.demo.example.org",
     "api" : {
       "type" : "openApi", //<2>
-      "use" : [ "open-api-file-reference" ] //<3>
+      "use" : [ "open-api-file-reference" ], //<3>
```

Comment:
Have we ensured that a file without "apiDefintionUrl" inside is also loaded as well - i am not sure here.